.github/scripts/js/constants.js

```diff
@@ -8,6 +11,7 @@ const labels = {
   'skip/copyright-validation': { type: 'skip-validation', validation_name: 'copyright' },
   'skip/markdown-validation': { type: 'skip-validation', validation_name: 'markdown' },
   'skip/actionlint': { type: 'skip-validation', validation_name: 'actionlint' },
+  [skipE2eLabel]: { type: 'skip-validation', validation_name: 'e2e_skip' },
```

Comment:
`type: 'skip-validation'` is for validation.yml workflow. It can break something. I think 'skip/e2e' can be used only as const.

Reply:
fixed